mysql-test/suite/innodb/t/percona_changed_page_bmp.test

```diff
@@ -255,6 +255,8 @@ DROP TABLE t3;
 --exec echo "restart:--innodb-data-home-dir=$MYSQLTEST_VARDIR/tmpdatadir" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
 --source include/wait_until_connected_again.inc
 
+select * from information_schema.innodb_changed_pages;
```

Comment:
Also, a more logically fitting place for this would be innodb_changed_pages.test testcase. But here we happen to have bug conditions set up for us, so OK